plugins/domains/d.pr.js

```diff
@@ -4,14 +4,11 @@ module.exports = {
         /^https?:\/\/(\w+\.)?d\.pr\/(?:i\/)?([a-zA-Z0-9]+)/i
     ],
 
-    // oembeds here to avoid redirects for 404s
     mixins: [
-        "oembed-thumbnail",
-        "oembed-site",
-        "oembed-title"
+        "*"
     ],
 
-    getLink: function(oembed) {
+    getLink: function(oembed, cheerio) {
```

Comment:
Finally, `cheerio` is the slowest parser out there. Let's try and use it only if the same data isn't available anywhere else. 

In your case, video src is given as `og.video` - so let's depend on `og` instead of cheerio